Model/Events/CartAdd.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Metrilo\Analytics\Model\Events;
+
+class CartAdd
```

Comment:
Please keep the names the same as the method calls (addToCart).
Otherwise 👍

Reply:
Are you 100% sure about this? Aren't they in different namespaces?

Reply:
Actually tested it again and i have made a mistake:
Initial attempt = use Metrilo\Analytics\Model\Events\AddToCart - definition with the same name already exists.
Correct way = use Metrilo\Analytics\Model\Events\AddToCart as CartAddEvent - proper way to include it.
Will rename the class.